Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -942,27 +866,27 @@ namespace OpenInfraPlatform {
 					const double b = t_shape->FlangeWidth * length_factor;
 					const double tw = t_shape->WebThickness * length_factor * 0.5;
 					const double tf = t_shape->FlangeThickness * length_factor;
-
-					double r1 = 0;
+					//Redo
+					double r1 = 0.0;
 					if(t_shape->FilletRadius) {
 						r1 = t_shape->FilletRadius * length_factor;
 					}
-
+					//Redo
 					double r2 = 0;
 					if(t_shape->FlangeEdgeRadius) {
 						r2 = t_shape->FlangeEdgeRadius * length_factor;
 					}
-
+					//Redo
 					double r3 = 0;
 					if(t_shape->WebEdgeRadius) {
 						r3 = t_shape->WebEdgeRadius * length_factor;
 					}
 					double fs = 0;
-
+					//Redo
 					if(t_shape->FlangeSlope) {
 						fs = t_shape->FlangeSlope * angle_factor;
 					}
-
+					//Redo
 					double ws = 0;
 					if(t_shape->WebSlope) {
 						ws = t_shape->WebSlope * angle_factor;

```

Comment:
This can also still be refactored using `value_or(...)`.

Reply:
Done